decidim-core/app/views/layouts/decidim/_application.html.erb

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html class="no-js">
   <head>
-    <title><%= decidim_page_title %></title>
+    <title><%= decidim_page_title.html_safe %></title>
```

Comment:
I'm not sure about this, we don't want HTML at the title right?

What about `CGI.unescapeHTML(title)` ?